tests/cluster/tests/29-slot-migration-replication.tcl

```diff
@@ -0,0 +1,351 @@
+#Tests that validate slot migration states are replicated propoerly
+
+source "../tests/includes/init-tests.tcl"
+source "../tests/includes/utils.tcl"
+
+variable master_count 3
+variable replica_count 1
```

Comment:
Usage of namespaces isn't very clear since you just reference them as if they were just normal variables.